src/deploy/functions/runtimes/node/index.ts

```diff
@@ -129,6 +129,18 @@ export class Delegate {
     env: backend.EnvironmentVariables
   ): Promise<backend.Backend> {
     if (previews.functionsv2) {
+      if (semver.valid(this.sdkVersion)) {
```

Comment:
Should this be

```js
if (!semver.valid(this.sdkVersion))
```

instead?

Reply:
Good catch, that's what i get for moving too quickly.